masonry_core/src/core/action.rs

```diff
@@ -28,7 +28,7 @@ pub enum Action {
     CheckboxToggled(bool),
     // FIXME - This is a huge hack
     /// Other.
-    Other(Box<dyn Any + Send>),
+    Other(Box<dyn AnyDebug + Send>),
```

Comment:
Self review comment for tomorrow: `Action` has a `Debug` implemenation, which needs to be updated to use this.